conformance_tests/sysman/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ add_subdirectory(test_sysman_events)
 add_subdirectory(test_sysman_memory)
 add_subdirectory(test_sysman_fabric)
 add_subdirectory(test_sysman_device)
+add_subdirectory(test_sysman_driver)
```

Comment:
I think we can append "_ext" or something in the name so it clearly indicates what kind of tests are involved.

Reply:
From spec point of view the API's are under https://spec.oneapi.io/level-zero/latest/sysman/api.html#driver. Hence test_sysman_driver looks fine for me. In test names, the extension information is mentioned